### PR TITLE
build

### PR DESCRIPTION
**Pull request by ckahub**
# Navigation
nav01

### CODE REVIEW

File: Components/Navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    background-color: transparent;
+    border-style:groove;
+    font-size: large;
+    font-family: Arial, Helvetica, sans-serif;   
+}
+
+ .left_Brand{
+    border: 1rem 1rem 1rem 1rem var(--default-color);    
+    border-radius: 2rem;
+    padding: 1rem 1rem 2rem 2rem;
```
Comment (SanketDhabarde) [inline Components/Navigation/navigation.css:16]: Can use `padding: 1rem 2rem` shorthand

File: Components/Navigation/navigation.css [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    width: 80%;
+    display: flex;
+    justify-content:first baseline;
+    align-items: strech;
+    /* box-shadow: 0rem 0.75rem 1rem var(--off-white); */
```
Comment (SanketDhabarde) [inline Components/Navigation/navigation.css:21]: don't keep unused code. you can remove it

**Review by SanketDhabarde — APPROVED** [review]
Looks good to me. Nicely written code

File: Components/Navigation/navigation.css [hunk 3]
```diff
@@ -0,0 +1,36 @@
+@import url('../../styles/utility.css');
+@import url('../../styles/navBar.css');
+@import url('../../styles/sideBar.css');
+@import url('../../styles/variable.css');
+
```
Comment (ishitakeshawani) [inline Components/Navigation/navigation.css:5]: You need to add a margin of 0 for the whole page in this file.

**Review by ishitakeshawani — APPROVED** [review]

**Pull request merged**